sourcecode/train.py

```diff
@@ -2,6 +2,8 @@
 
 import argparse
 
+import os
+import sagemaker_containers
```

Comment:
I am not sure how this works, since the package is not used anywhere in the rest of the file. I guess this import directly takes care of all the bootstrapping with SageMaker via the use of environment variables? Could you please apply a check whether this package exists (make a conditional import)? That way, the code could be run on local machines as well as SM.